mono/mini/Makefile.am.in

```diff
@@ -1,6 +1,8 @@
 count=100000
 mtest=for_loop
 monodir=$(top_builddir)
+mono=$(if $(MONO_EXECUTABLE),$(MONO_EXECUTABLE),mono)
+sep=$(if $(LIST_SEPARATOR),$(LIST_SEPARATOR),:)
```

Comment:
In the mcs/ build system this is called `$(PLATFORM_PATH_SEPARATOR)` (just for sake of consistency).